qualtran/bloqs/factoring/mod_exp_test.py

```diff
@@ -85,6 +85,12 @@ def generalize(b: Bloq) -> Optional[Bloq]:
     assert counts1 == counts2
 
 
+def test_mod_exp_t_complexity():
+    bloq = ModExp(base=8, exp_bitsize=3, x_bitsize=10, mod=50)
+    tcomp = bloq.t_complexity()
+    assert tcomp.t > 0
```

Comment:
do we know what value it should be?

Reply:
I put this in because somewhere during the change the `t_complexity` method just stopped functioning (due to the reorganization). Having `assert_equivalent_bloq_counts` the whole way down will provide the coverage for making sure the estimates are correct